app/grandchallenge/publications/templates/publications/publication_list.html

```diff
@@ -0,0 +1,51 @@
+{% extends "base.html" %}
+{% load static %}
+{% load publication_extras %}
+{% load guardian_tags %}
+{% load url %}
+
+
+{% block title %}Publications - {{ block.super }}{% endblock %}
+
+{% block breadcrumbs %}
+    <ol class="breadcrumb">
+        <li class="breadcrumb-item active" aria-current="page">Publications</li>
+    </ol>
+{% endblock %}
+
+{% block content %}
+        {% include "grandchallenge/partials/filters.html" with filter=filter filters_applied=filters_applied %}
+    <div>
+        <h2>Publications</h2>
+    </div>
+    <div class="d-flex justify-content-between my-3">
+        <h6 class="ml-1 mt-2">{{ num_publications }} publications | {{ num_citations }} citations </h6>
+        <a class="btn btn-primary" href="{% url 'publications:create' %}" role="button">Add a publication</a>
+    </div>
+        <ul class="list-group">
+            {% for publication in object_list %}
+                <li class="list-group-item p-0">
+                    <div class="py-2 pl-3">
+                        {{ publication.citation|safe }}
+                    </div>
+                    <div class="pt-1 pb-2 pl-3">
+                        <a class="btn btn-outline-primary btn-sm mb-1 mr-1" href="{{ publication.url }}"><i class="fa fa-link"></i></a>
+                        {% with publication|get_associated_objects as publication_objects %}
+                            {% if publication_objects %}
+                                {% for obj, obj_model in publication_objects.items %}
+                                    {% get_obj_perms request.user for obj as "object_perms" %}
+                                    {% if "view_"|add:obj_model in object_perms %}

```

Comment:
This is probably going to make a lot of requests. See the Organization Detail view about how to use an ObjectPermissionsChecker to reduce these.

Reply:
I believe I'm following the Organization Detail view as closely as possible already. The only difference is that in this case here, we need to construct the permission string based on the object we're trying to display, but this `"view_"|add:obj_model` doesn't do any extra requests, so it should be fine, I think/

Reply:
I think the `get_object_perms` is the line which will do the extra calls right now.

Reply:
I see what is happening now, you're right! I suggested another approach in a different comment.